lib/public/views/Runs/Overview/RunsOverviewModel.js

```diff
@@ -842,19 +732,28 @@ export class RunsOverviewModel extends OverviewPageModel {
     /**
      * Return the model handling the filtering on run types
      *
-     * @return {EorReasonsFilterModel} the run type filtering model
+     * @return {EorReasonFilterModel} the run type filtering model
      */
     get eorReasonsFilterModel() {
         return this._eorReasonsFilterModel;
```

Comment:
I think that you can also rename the field to singular

Reply:
I just fixed the docblock, this would be a bit too much refactoring